docs/admin/auth/methods.rst

```diff
@@ -132,10 +132,16 @@ Token must contain the following claims:
 
 ``username`` - user name in a third party app.
 
-``iss``, ``username`` and ``aud`` values must match the values created by
-``CREATE USER`` statement. If ``aud`` has not been defined on the
-``CREATE USER`` statement, the cluster id is used and must match the token's
-``aud`` value. See :ref:`create-user-jwt` for details.
+The token's ``iss``, ``username`` and ``aud`` values must match the
+corresponding values within CrateDB.
+
+CrateDB uses :ref:`jwt_defaults` if they are defined or values defined on a
+CrateDB uses the cluster-wide :ref:`jwt_defaults` if they are configured. If no cluster defaults are defined, it will use the values specified at the individual user level via the ``CREATE USER`` statement.
```

Comment:
I don't understand this, what is meant with `... or values defined on a CrateDB uses the cluster-wide...`?

Reply:
I applied suggestion from https://github.com/crate/crate/pull/16763#discussion_r1810806345 and forgot to rework it, thanks for catching!

Also, this is actually a node setting ([see comment](https://github.com/crate/crate/pull/16763#discussion_r1799638621)), so I removed "cluster-wide" part. 
cc @henrikingo, @proddata.

Moreover, we can use default only if "iss" is defined. 

Also, providing default `aud` is not enough to make default usable (see `aud` descritption in node.rst)

I rephrased it.